tox.ini

```diff
@@ -10,3 +12,11 @@ deps =
 commands =
   pytest --ignore setup.py {posargs}
   pytest urlextract {posargs: --doctest-modules}
+
+[testenv:black]
+basepython = python3.6
+deps =
+  black
+commands =
+  black urlextract --skip-string-normalization
+  black tests --skip-string-normalization
```

Comment:
I have just small thing. I checked the tests for black and I saw `1 file reformatted, 2 files left unchanged.` Could you just add flag to both black commands that it only checks the file format but does not modify it (should be something like `--check`).

Reply:
ok @lipoja

Reply:
@lipoja added! Does this mean, we should revert the black-formatted files in `urlextract` and `tests` folder?

Reply:
No, we should keep them re-formatted in this PR otherwise the CI will fail.